packages/runner-ct/package.json

```diff
@@ -8,6 +8,8 @@
     "build": "webpack",
     "build-prod": "cross-env NODE_ENV=production yarn build && tsc",
     "clean-deps": "rm -rf node_modules",
+    "cypress:open": "ts-node ../../scripts/start.js --component-testing --project ${PWD}",
```

Comment:
I wonder how strict we want to be? We could add an additional `--transpile-only` flag to ignore and TS errors. But I am fine with this.